client/src/App.jsx

```diff
@@ -19,27 +22,25 @@ function App() {
   function myFunction() {
     let question = prompt("Please enter your question");
 
-    axios.post("http://127.0.0.1:5001/POST", {
-      question: question,
-    })
-    .then(function (response) {
-      console.log(JSON.stringify(response));
-    })
-    .catch(function (error) {
-      console.log(JSON.stringify(error));
-    });
-  
-    if (question != null) {
-      document.getElementById("post").innerHTML =
-        "good question, let me think...";
-    }
+    axios
```

Comment:
Another way to write is using the `async`/`await` syntax similar to above `GET` request.